cmd/crypto/header.go

```diff
@@ -66,8 +66,7 @@ type s3 struct{}
 // IsRequested returns true if the HTTP headers indicates that
 // the S3 client requests SSE-S3.
 func (s3) IsRequested(h http.Header) bool {
-	_, ok := h[SSEHeader]
-	return ok
+	return h.Get(SSEHeader) != ""
```

Comment:
Not correct. The header can be set but be empty.

Reply:
removed this.